as_perimenoun_oi_gynaikes.ass

```diff
@@ -556,47 +557,47 @@ Dialogue: 0,0:41:08.99,0:41:10.76,Default,,0,0,0,,Is it the refs fault \Nif you
 Dialogue: 0,0:41:10.76,0:41:13.56,Default,,0,0,0,,- Was there a score reversal at the last minute?\N- Three meters from the penalty box?!
 Dialogue: 0,0:41:13.56,0:41:15.99,Default,,0,0,0,,Are you kidding me?\N Manolas tackles Skartados...
 Dialogue: 0,0:41:15.99,0:41:18.64,Default,,0,0,0,,...sends him airborne for three meters...
-Dialogue: 0,0:41:18.64,0:41:21.09,Default,,0,0,0,,...and falls in the penatly box!\N - He was faking it!
+Dialogue: 0,0:41:18.64,0:41:21.09,Default,,0,0,0,,...and falls in the penatly box!\N - Was he a bird?!
 Dialogue: 0,0:41:21.09,0:41:24.84,Default,,0,0,0,,And the boy is lying half-dead in the mud,\Nand the refs shows him the yellow card...
-Dialogue: 0,0:41:24.84,0:41:27.09,Default,,0,0,0,,...as if he is faking it? And he still has a heartbeat!?
+Dialogue: 0,0:41:24.84,0:41:27.09,Default,,0,0,0,,...as if he is faking it? And he is still lives!?

```

Comment:
Syntactically wrong and not an improvement.

Reply:
Yeap, my fault, I would prefer "And he is still alive!?" / "And he is still left alive!?"
'left alive' implies that the PAOK fans should have killed the referee for being 'injust'

Reply:
The original translation implies the same thing with an edgy expression. I don't feel that this should change.